### PR TITLE
DOC Rename validator classes

### PR DESCRIPTION
**Pull request by emteknetnz**
Issue https://github.com/silverstripe/silverstripe-framework/issues/10908

- [x] Have updated any merged up docs changes to old class names e.g. https://github.com/silverstripe/developer-docs/pull/647#discussion_r1872017222

### CODE REVIEW

File: en/08_Changelogs/6.0.0.md [hunk 1]
```diff
@@ -898,6 +898,14 @@ The following classes have been changed when code was moved to the `silverstripe
 |`SilverStripe\ExternalLinks\Tasks\CurlLinkChecker`|[`SilverStripe\Reports\ExternalLinks\Tasks\CurlLinkChecker`](api:SilverStripe\Reports\ExternalLinks\Tasks\CurlLinkChecker)|
 |`SilverStripe\ExternalLinks\Tasks\LinkChecker`|[`SilverStripe\Reports\ExternalLinks\Tasks\LinkChecker`](api:SilverStripe\Reports\ExternalLinks\Tasks\LinkChecker)|
 
+The following subclasses of [`Validator`](api:SilverStripe\Forms\Validator) have been renamed:
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md]: That's a link to the old FQCN and won't work here.

File: en/08_Changelogs/6.0.0.md [hunk 2]
```diff
@@ -898,6 +898,14 @@ The following classes have been changed when code was moved to the `silverstripe
...
+
+|Old FQCN|New FQCN|
+|---|---|
+|`SilverStripe\Forms\RequiredFields`|[`SilverStripe\Forms\RequiredFieldsValidator`](api:SilverStripe\Forms\RequiredFieldsValidator)|
```
Comment (ChloeSartorelli) [inline en/08_Changelogs/6.0.0.md]: Need to update the namespace and add the `CompositeValidator`.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

File: en/08_Changelogs/6.0.0.md (hunk 1 above)
Comment (emteknetnz) [reply en/08_Changelogs/6.0.0.md]: Updated

File: en/08_Changelogs/6.0.0.md (hunk 2 above)
Comment (emteknetnz) [reply en/08_Changelogs/6.0.0.md]: Updated

Comment (ChloeSartorelli) [reply en/08_Changelogs/6.0.0.md]: Still not done?? You added `CompositeValidator` but didn't update the namespaces of these two.

**Review by GuySartorelli — CHANGES_REQUESTED** [review]

**Comment by GuySartorelli** [comment]
Linting failure

File: en/08_Changelogs/6.0.0.md (hunk 2 above)
Comment (emteknetnz) [reply en/08_Changelogs/6.0.0.md]: Updated

**Review by GuySartorelli — APPROVED** [review]
LGTM

**Pull request merged**